src/map/packet_system.cpp

```diff
@@ -4855,7 +4855,7 @@ void SmallPacket0x077(map_session_data_t* const PSession, CCharEntity* const PCh
     {
         case 0: // party
         {
-            if (PChar->PParty != nullptr && PChar->PParty->GetLeader() == PChar)
+            if (PChar && PChar->PParty != nullptr && PChar->PParty->GetLeader() == PChar)
```

Comment:
Same here, we can check PChar once before the switch and then continue afterwards with impunity